### ISSUE TITLE
Indexes in prisma schema

### ISSUE THREAD

**Issue by ojeytonwilliams** [freeCodeCamp/freeCodeCamp#51688]
Do we have a plan for how to keep the prisma schema up-to-date with any indexes we create in production? This only matters if the indexes affect how the database behaves, i.e. they're unique, but we should think about how (or if) we're going to keep the schema up to date.

Periodic `prisma db pull`s?

**Comment by ShaunSHamilton** [comment]
`prisma db pull` only works if we have a single schema across the whole db. Even after a decent normalization, this is unlikely.

**Comment by ojeytonwilliams** [comment]
Well, finding out that the schema has drifted is useful info. However, for indexes, I think you can just directly query for them with https://www.mongodb.com/docs/manual/reference/command/listIndexes/#mongodb-dbcommand-dbcmd.listIndexes I.e. there's no need to sample.

I've not confirmed this, but it seems likely that that's what `prisma db pull` does.

**Comment by ShaunSHamilton** [comment]
> Well, finding out that the schema has drifted is useful info

Right, but we won't get this with `prisma db pull`. Best as I can remember, it will only look at `1_000` docs to generate the models.

**Comment by ahmaxed** [comment]
How about initially keeping the indexes in sync with [listIndexes](https://www.mongodb.com/docs/manual/reference/command/listIndexes/#mongodb-dbcommand-dbcmd.listIndexes)?

**Comment by anishlukk123** [comment]
do we have any plans to do this and if we do how do we get priortized

**Comment by jdwilkin4** [comment]
As discussed in the staff chat, I am closing this due to inactivity.

If you wish to reopen it and work on, then feel free to reopen 👍🏾
